library/core/src/num/f16.rs

```diff
@@ -267,7 +267,7 @@ impl f16 {
     ///
     /// ```
     /// #![feature(f16)]
-    /// # #[cfg(target_arch = "aarch64")] { // FIXME(f16_F128): rust-lang/rust#123885
```

Comment:
No strong reason, just a slightly smaller `cfg` (gets copied around a lot) and to be consistent with the rest of the file and `f128`. The `core` doctests are more or less just for smoke, since `std` has the better tests and all the platform config logic. (Unfortunately there doesn't seem to be an easy way to share this config with `core`).

BUT this made me double check the other gates in this file, and I realize I forgot to restrict to only linux. Updated that.